war/src/main/webapp/WEB-INF/elasticsearch.yml

```diff
@@ -31,39 +33,28 @@ path.home: es
 #
 path.data: ${exo.data.dir}
 
-# Path to where plugins are installed:
-#
-path.plugins: es/plugins
-
 # Default usage of es embedded addon is for test
 #
-index.number_of_replicas: 0
+#index.number_of_replicas: 0
```

Comment:
@fdrouet Since 5.0, the index.* properties (except index.codec) are not allowed anymore on the global configuration. they must be declared in the index templates. https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_50_settings_changes.html